<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="0" skipped="1">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-05-13T12:49:40 IST" name="Default suite" finished-at="2022-05-13T12:49:40 IST" duration-ms="79">
    <groups>
    </groups>
    <test started-at="2022-05-13T12:49:40 IST" name="Default test" finished-at="2022-05-13T12:49:40 IST" duration-ms="79">
      <class name="com.crm.comcast.OrgTest.Test.DataProviderPracticeOrganizationTest">
        <test-method is-config="true" signature="openDataBaseConnection()[pri:0, instance:com.crm.comcast.OrgTest.Test.DataProviderPracticeOrganizationTest@5d7148e2]" started-at="2022-05-13T12:49:39 IST" name="openDataBaseConnection" finished-at="2022-05-13T12:49:40 IST" duration-ms="1232" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openDataBaseConnection -->
        <test-method is-config="true" signature="launchApplicationWindow()[pri:0, instance:com.crm.comcast.OrgTest.Test.DataProviderPracticeOrganizationTest@5d7148e2]" started-at="2022-05-13T12:49:40 IST" name="launchApplicationWindow" finished-at="2022-05-13T12:49:40 IST" duration-ms="1" status="FAIL">
          <exception class="java.io.FileNotFoundException">
            <message>
              <![CDATA[.\data\commondata.properties (The system cannot find the path specified)]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.io.FileNotFoundException: .\data\commondata.properties (The system cannot find the path specified)
at java.base/java.io.FileInputStream.open0(Native Method)
at java.base/java.io.FileInputStream.open(FileInputStream.java:216)
at java.base/java.io.FileInputStream.<init>(FileInputStream.java:157)
at java.base/java.io.FileInputStream.<init>(FileInputStream.java:111)
at com.crm.comcast.GenericUtility.FileUtitlty.getKeyValue(FileUtitlty.java:20)
at com.crm.comcast.GenericUtility.BaseAnnotaionConfigClass.launchApplicationWindow(BaseAnnotaionConfigClass.java:71)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:176)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:122)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.io.FileNotFoundException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplicationWindow -->
        <test-method signature="createOrganizationUsingDataProviderTest(java.lang.String,java.lang.String)[pri:0, instance:com.crm.comcast.OrgTest.Test.DataProviderPracticeOrganizationTest@5d7148e2]" started-at="2022-05-13T12:49:40 IST" name="createOrganizationUsingDataProviderTest" data-provider="data" finished-at="2022-05-13T12:49:40 IST" duration-ms="0" status="SKIP">
          <exception class="java.lang.RuntimeException">
            <message>
              <![CDATA[java.io.FileNotFoundException: .\data\Testdata.xlsx (The system cannot find the path specified)]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.RuntimeException: java.io.FileNotFoundException: .\data\Testdata.xlsx (The system cannot find the path specified)
at org.testng.internal.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:50)
at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:146)
at org.testng.internal.Parameters.handleParameters(Parameters.java:798)
at org.testng.internal.Parameters.handleParameters(Parameters.java:740)
at org.testng.internal.ParameterHandler.handleParameters(ParameterHandler.java:59)
at org.testng.internal.ParameterHandler.createParameters(ParameterHandler.java:38)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:791)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.io.FileNotFoundException: .\data\Testdata.xlsx (The system cannot find the path specified)
at java.base/java.io.FileInputStream.open0(Native Method)
at java.base/java.io.FileInputStream.open(FileInputStream.java:216)
at java.base/java.io.FileInputStream.<init>(FileInputStream.java:157)
at java.base/java.io.FileInputStream.<init>(FileInputStream.java:111)
at com.crm.comcast.GenericUtility.ExcelUtitlity.getDataUsingDataProvider(ExcelUtitlity.java:84)
at com.crm.comcast.OrgTest.Test.DataProviderPracticeOrganizationTest.getData(DataProviderPracticeOrganizationTest.java:43)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:77)
at org.testng.internal.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:46)
... 25 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.RuntimeException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createOrganizationUsingDataProviderTest -->
        <test-method is-config="true" signature="closeApplicationWindow()[pri:0, instance:com.crm.comcast.OrgTest.Test.DataProviderPracticeOrganizationTest@5d7148e2]" started-at="2022-05-13T12:49:40 IST" name="closeApplicationWindow" finished-at="1970-01-01T05:30:00 IST" duration-ms="-1652426380509" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeApplicationWindow -->
        <test-method is-config="true" signature="closeBrowser()[pri:0, instance:com.crm.comcast.OrgTest.Test.DataProviderPracticeOrganizationTest@5d7148e2]" started-at="2022-05-13T12:49:40 IST" name="closeBrowser" finished-at="2022-05-13T12:49:40 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
        <test-method is-config="true" signature="closeDataBaseConnection()[pri:0, instance:com.crm.comcast.OrgTest.Test.DataProviderPracticeOrganizationTest@5d7148e2]" started-at="2022-05-13T12:49:40 IST" name="closeDataBaseConnection" finished-at="2022-05-13T12:49:40 IST" duration-ms="40" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeDataBaseConnection -->
      </class> <!-- com.crm.comcast.OrgTest.Test.DataProviderPracticeOrganizationTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
